src/main/app/controller/courts/CasesHeardController.ts

```diff
@@ -0,0 +1,80 @@
+import autobind from 'autobind-decorator';
+import {AuthedRequest} from '../../../types/AuthedRequest';
+import {Response} from 'express';
+import {AreaOfLaw} from '../../../types/AreaOfLaw';
+import {Error} from '../../../types/Error';
+import {CasesHeardPageData} from '../../../types/CasesHeardPageData';
+import {CSRF} from '../../../modules/csrf';
+import {AxiosError} from 'axios';
+
+@autobind
+export class CasesHeardController {
+
+  getAreasOfLawErrorMsg = 'A problem occurred when retrieving the areas of law. ';
+  getCourtAreasOfLawErrorMsg = 'A problem occurred when retrieving the court areas of law. ';
+  putCourtAreasOfLawErrorMsg = 'A problem occurred when updating the court areas of law. ';
+
+  public async get(req: AuthedRequest, res: Response): Promise<void> {
+    await this.render(req, res, [], false, true, null, null);
```

Comment:
If you gave the last parameter of the render method a default value, you could just call `this.render(req, res)` here.